.hermes/conveyor/work-1efc01c2/adr.md

```
@@ -0,0 +1,77 @@
+# ADR — Formalize print-statement lint enforcement in perl-lsp-launcher
+
+## Status
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)